hack/jenkins/e2e.sh

```diff
@@ -645,6 +645,20 @@ case ${JOB_NAME} in
     : ${USE_KUBEMARK:="true"}
     : ${NUM_MINIONS:="10"}
     ;;
+
+  kubernetes-kubemark-gce-scale)
+    : ${E2E_CLUSTER_NAME:="kubernetes-kubemark-scale"}
+    : ${E2E_NETWORK:="kubernetes-kubemark-scale"}
+    : ${PROJECT:="kubernetes-scalability"}
```

Comment:
It is a new project, but I did indeed mistype it.  It should be kubernetes-scale.  Fixing now.